### PR TITLE
Draft minutes for Packaging WG meeting 21/11/18

### PR DESCRIPTION
**Pull request by drbenmorgan**
Filled out from live notes, plus some minor grammar/spelling
corrections.

@amadio, I couldn't add you as a direct reviewer, but could you check for any mistakes in the description of Gentoo/Portage please?

### CODE REVIEW

File: organization/_posts/2018/2018-11-21-packaging.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  - Gentoo Prefix?
+- Can use Gentoo Prefix via CVMFS, available prefixes for:
+  - Native Linux (distro independent as Gentoo ships own glibc), macOS
+  - Docker container using host CVMFS
+  - Includes very lightweight images like Busybox (1MB), then use tooling from prefix
+- Documentation links: getting started on bare metal, Gentoo Linux image, just Gentoo Prefix on other Linuces, macOS etc
+  - "Test drive" on HSF repo on get started with Prefix
+- Further guides on the basics of using Portage
+  - Installing, configuration etc, plus associated tooling to help with packaging processes
+- Basic installation of packages via `emerge` program, eg. `emerge <args> package`
```
Comment (graeme-a-stewart) [inline organization/_posts/2018/2018-11-21-packaging.md]: e.g. (sorry, super picky mode was on)

**Comment by amadio** [comment]
Sure, I had a look yesterday on Google Docs, and it seems fine. I will let you know if there's anything to correct.

**Comment by graeme-a-stewart** [comment]
URL checker failures all look spurious to me, nothing bad in this PR

**Comment by amadio** [comment]
No problems I could see. I uploaded the slides yesterday after the meeting, just FYI. Maybe you want to link in the notes.

**Comment by graeme-a-stewart** [comment]
> Maybe you want to link in the notes.

There's always a link to Indico at the top of the minutes

**Pull request merged**